app/src/main/java/org/schabi/newpipe/error/ReCaptchaActivity.java

```diff
@@ -162,6 +162,9 @@ private void saveCookiesAndFinish() {
             setResult(RESULT_OK);
         }
 
+        // Navigate to blank page (unloads youtube to prevent background playback)
+        recaptchaBinding.reCaptchaWebView.loadData("", "text/html", null);
```

Comment:
According to [this SO](https://stackoverflow.com/questions/3715482/how-to-clear-webview-content) you should use
```suggestion
        recaptchaBinding.reCaptchaWebView.loadUrl("about:blank");
```